frontend/public/components/service-instance.tsx

```diff
@@ -0,0 +1,229 @@
+      </div>
+      <div className="co-m-table-grid__body">
+        {rows}
+      </div>
+    </div>
+  </div>;
+};
+
+const ServiceInstanceParameters: React.SFC<ServiceInstanceParametersProps> = ({parameters}) => {
+  return <div className="co-m-pane__body">
+    <SectionHeading text="Parameters" />
+    <dl className="co-m-resource__details">
+      {_.map(parameters, (v, k) => <React.Fragment key={k}>
+        <dt>{k}</dt>
+        <dd>{v}</dd>
+      </React.Fragment>
+      )}
+    </dl>
+  </div>;
+};
+
+const ServiceInstanceDetails: React.SFC<ServiceInstanceDetailsProps> = ({obj: si}) => {
+  const statusMessage = serviceInstanceStatusMessage(si);
+  const plan = si.spec.clusterServicePlanExternalName || si.spec.servicePlanExternalName;
+  const parameters = si.status.externalProperties.parameters;
+
+  return <React.Fragment>
+    <div className="co-m-pane__body">
+      <SectionHeading text="Service Instance Overview" />
+      <div className="row">
+        <div className="col-sm-6">
+          <ResourceSummary resource={si} showPodSelector={false} showNodeSelector={false} />
+        </div>
+        <div className="col-sm-6">
+          <dl className="co-m-pane__details">
+            <dt>External Name</dt>
+            {/* TODO:  add link */}
+            <dd>{si.spec.clusterServiceClassExternalName}</dd>
+            <dt>Status</dt>
+            <dd><ServiceInstanceStatus obj={si} /></dd>
+            {statusMessage && <dt>Status Message</dt>}
+            {statusMessage && <dd>{statusMessage}</dd>}
+            <dt>Plan</dt>
+            <dd>{plan ? plan : '-'}</dd>
+          </dl>
+        </div>
+      </div>
+    </div>
+    <div className="co-m-pane__body">
+      <SectionHeading text="Conditions" />
+      <Conditions conditions={si.status.conditions} />
+    </div>
+    {!_.isEmpty(si.spec.parametersFrom) && <ServiceInstanceParametersSecrets obj={si} /> }
+    {!_.isEmpty(parameters) && <ServiceInstanceParameters parameters={parameters} /> }
+  </React.Fragment>;
+};
+
+const pages = [navFactory.details(ServiceInstanceDetails), navFactory.editYaml(), navFactory.events(ResourceEventStream)];
+export const ServiceInstanceDetailsPage: React.SFC<ServiceInstanceDetailsPageProps> = props =>
+  <DetailsPage
+    {...props}
+    kind={ServiceInstancesReference}
+    menuActions={menuActions}
+    pages={pages} />;
+ServiceInstanceDetailsPage.displayName = 'ServiceInstanceDetailsPage';
+
+const ServiceInstancesHeader = props => <ListHeader>
+  <ColHead {...props} className="col-md-3 col-sm-4 col-xs-6" sortField="metadata.name">Name</ColHead>
+  <ColHead {...props} className="col-md-3 col-sm-4 col-xs-6" sortField="metadata.namespace">Namespace</ColHead>
+  <ColHead {...props} className="col-md-3 col-sm-4 hidden-xs" sortField="status.conditions">Status</ColHead>
+  <ColHead {...props} className="col-md-3 hidden-sm hidden-xs" sortField="metadata.creationTimestamp">Created</ColHead>
+</ListHeader>;
```

Comment:
We probably want to add a service class link in the table when we have a page to link to. Fine to add later.

Plan would be good to show here as well.